TTS/bin/distribute.py

```diff
@@ -23,7 +23,7 @@ def main():
     group_id = time.strftime("%Y_%m_%d-%H%M%S")
 
     assert num_gpus > 1, 'distributed.py requires multiple available GPUs'
-    visible_gpus = os.environ['CUDA_VISIBLE_DEVICES'] if 'CUDA_VISIBLE_DEVICES' in os.environ else list(range(0, num_gpus))
```

Comment:
Good catch! Would it be easier to avoid the split() below altogether and generate lists right here? Something like 
`visible_gpus = list(os.environ['CUDA_VISIBLE_DEVICES']) if 'CUDA_VISIBLE_DEVICES' in os.environ else list(range(0, num_gpus))`

Reply:
Good thought, but no unfortunately, you need the split somewhere. If you do what you're proposing you'd simply get a list of all characters in the string (including commas and whitespace).